twitter/twitterapp/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.db.models.base import Model
+
+# Create your models here.
+class User(models.Model): 
```

Comment:
Add a class level docstring to document what this model represents.

For example, something like:

```
class User(models.Model): 
    """User represents an account in our system.

    A user is an owner of the tweets in the system and is considered
    as the author. Each person who signs up has a corresponding 
    User in the database.
    """

```